src/data/actions/courseInfo.js

```diff
@@ -20,6 +23,20 @@ export function requestCourseInfo(id) {
   return { type: REQUEST_COURSE_INFO, id };
 }
 
+export function createNewCourse(courseData) {
+  return {
+    type: CREATE_COURSE, courseData,
+  };
+}
```

Comment:
I don't care, really. But this is a different formatting than others.

Reply:
I would just change them all the match the constant you're using.
createCourseInfo (or something like that. Mine is editCourseInfo)
createCourseSuccess
createCourseFail